### PR TITLE
Pub 1275 a z changes

### PR DESCRIPTION
**Pull request by kurt-uk**
### JIRA link ###
https://tools.hmcts.net/jira/browse/PUB-1275

### Change description ###
branched from 1264

Updated a-z pages on public and subscribe journey, removed SJP hack and all locations display under the same collection. headings removed.

also applied filter heading change on subscribe a-z page

## screenshots
Screenshot 2022-05-16 at 15 19 42
Screenshot 2022-05-16 at 15 19 58

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by joshblackmoor — DISMISSED** [review]

**Review by ChrisS1512 — CHANGES_REQUESTED** [review]
Looks good! Unit tests failed but I think it could just be the environment. Could you have a look?

**Comment by kurt-uk** [comment]
> Looks good! Unit tests failed but I think it could just be the environment. Could you have a look?

@ChrisS1512 This build wont pass e2e until BE is merged as its using new api of `/location...` rather than courts

**Review by ChrisS1512 — DISMISSED** [review]

**Comment by kurt-uk** [comment]
Holding off merging until all BE related have been merged, then e2e need looking at, but moving across to allow testing to progress locally

**Review by joshblackmoor — APPROVED** [review]

**Pull request merged**
